src/widgets/HeaderRight.js

```diff
@@ -5,30 +5,11 @@
 
 import React from 'react';
 
-import PropTypes from 'prop-types';
-
-import { connect } from 'react-redux';
-
-import { selectUsingVaccines } from '../selectors/modules';
 import { SyncState } from './SyncState';
 import { FlexRow } from './FlexRow';
-import { TemperatureSyncState } from './TemperatureSyncState';
-
-const mapStateToProps = state => {
-  const usingVaccines = selectUsingVaccines(state);
 
-  return { usingVaccines };
-};
-
-const HeaderRightComponent = ({ usingVaccines }) => (
+export const HeaderRight = () => (
```

Comment:
just curious - the sensor sync bits from the header are removed and not returning?
There's no manual sync option now?

Reply:
Yeah. Majority one that it didn't look good in the header and also that manual downloading of temperatures wasn't required 🤷